src/main/java/com/hotels/beeju/BeejuJUnitRule.java

```diff
@@ -132,29 +90,29 @@ protected void after() {
    * @return the name of the JDBC driver class used to access the database.
    */
   public String driverClassName() {
-    return driverClassName;
-  }
-
-  /**
-   * @return a copy of the {@link HiveConf} used to create the Hive Metastore database. This {@link HiveConf} should be
-   *         used by tests wishing to connect to the database.
-   */
-  public HiveConf conf() {
-    return new HiveConf(conf);
+    return core.driverClassName();
   }
 
   /**
    * @return the name of the pre-created database.
    */
   public String databaseName() {
-    return databaseName;
+    return core.databaseName();
   }
 
   /**
    * @return the JDBC connection URL to the HSQLDB in-memory database.
    */
   public String connectionURL() {
-    return connectionURL;
+    return core.connectionURL();
+  }
+
+  /**
+   * @return a copy of the {@link HiveConf} used to create the Hive Metastore database. This {@link HiveConf} should be
```

Comment:
this is not valid javadoc for this method anymore, Maybe just link to the BeejuCore method: `@see com.my.package.Class#method()`

Reply:
can you change the javadoc here?